installation/nuget.md

```diff
@@ -142,28 +142,36 @@ You can [generate your Telerik NuGet API Key on telerik.com](https://www.telerik
 
 The `Telerik.UI.for.Blazor` NuGet package and most of its dependencies reside on `nuget.telerik.com`. On the other hand, the [Telerik icon packages]({%slug common-features-icons%}) reside on `nuget.org`. The correct [package source mapping](https://learn.microsoft.com/en-us/nuget/consume-packages/package-source-mapping) configuration should be similar to:
 
+> Make sure that the `key` values in the `packageSourceMapping` section match the `key` values in the `packageSources` section, otherwise you will get a "Package not found" error.
```

Comment:
Our style guide [recommends](https://docs.telerik.com/style-guide/element-formatting) code formatting for error messages. 

```suggestion
> Make sure that the `key` values in the `packageSourceMapping` section match the `key` values in the `packageSources` section, otherwise you will get a `Package not found` error.
```